### ISSUE TITLE
how can I remove caprine to install a new version

### ISSUE THREAD

**Issue by violette-de-montgazin** [sindresorhus/caprine#1320]
<!-- I downloaded caprine on my Ubuntu 18.04 LTS from this site https://www.ubuntufree.com/download-facebook-messenger-for-ubuntu/#Caprine_Features but I can't get video calls. I get it from your forum that it could be facebook's will to disable it every time new solutions are found.
 I would like to disinstall it (caprine) but I  can't,  even in a terminal. Can you help me do that, please?
Violette

---

Caprine 2.44.0
Electron 8.0.0
linux 4.15.0-91-generic
Locale: en-US

**Comment by violette-de-montgazin** [comment]
It seems my description has disappeared! 
I downloaded caprine on my Ubuntu 18.04 LTS from the website:  https://www.ubuntufree.com/download-facebook-messenger-for-ubuntu/#Caprine_Features. 
I can't make video calls so I would like to disinstall it but I can't, even with a terminal!
Can you help me do that please?

**Comment by botex98** [comment]
I would suggest uninstalling with AppCleaner or CleanMyMacX. I have done it before to uninstall Caprine for the same reason.

**Comment by violette-de-montgazin** [comment]
Thank you for your quick answer! But aren't they just for Mac computers? I can't find any of them in my Ubuntu software center.

**Comment by botex98** [comment]
Yes they are for mac. But I was reffering to the fact that these applications don't just remove the .app file but the configuration files as well. Take a look at this [site](https://www.tecmint.com/ccleaner-alternatives-for-ubuntu/)

**Comment by violette-de-montgazin** [comment]
Oh yes! I'll take a closer look at these apps tomorrow morning, after a good night's sleep (it's time to go to bed in Bordeaux!). Thank you

**Comment by violette-de-montgazin** [comment]
![term caprine1](https://user-images.githubusercontent.com/62699345/78337836-f3d8c980-7591-11ea-9a73-ee3e690abab1.png)
Here is what has something to do with caprine (attachment). I intend to get rid of every item using the command "rm" in the terminal. Is it a good idea? What do you think? Tks
